exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Database.kt

```diff
@@ -175,8 +175,8 @@ class Database private constructor(
         fun connect(
             url: String,
             driver: String = getDriver(url),
-            user: String = "",
-            password: String = "",
+            user: String = "root",
+            password: String = "Exposed_password_1!",
```

Comment:
Are these meant to be left in as default arguments for the public `Database.connect()`?

Reply:
nope, will move to the test one